src/list.ts

```diff
@@ -0,0 +1,14 @@
+import { HTML } from "@bigtest/interactor";
+
+export const List = HTML.extend<HTMLElement>("MUI List")
+  .locator((element) => element.getAttribute("aria-label") ?? element.innerText)
+  .selector('[class*="MuiList-root"]');
+
+export const ListItem = HTML.extend<HTMLElement>("MUI ListItem")
+  .selector('[class*="MuiListItem-root"]')
+  .filters({
+    disabled: {
+      apply: (element) => element.hasAttribute("disabled") || element.getAttribute("aria-disabled") == "true",
+      default: false,
+    },
+  });
```

Comment:
In most of the `collection/item` interfaces, we include an `index` filter that lets us choose an item based on where it appears in its collection. E.g.

```js
await ListItem({ index: 0 }).has({ text: "One" });
```

Reply:
It requires to make core changes. The filter function doesn't know about a context and we traverse through elements from query selection. It's a good suggestion. Let's create an issue, to think a little more about it.

Reply:
The easiest way is look to the parent, so we need know what parent we looking for. And then count children with one common parent. We need to think how to interpret it for error message if we found duplicates.

Hm... we can try extract parent from ancestors list. But it's not obvious. Too many edge cases.

Reply:
Does it need a core change? I was more thinking that we can use `element.closest()`  and then compare it with the children. It's not an elegant implementation, but it should work until we can figure out the best change to core to make it work.

Reply:
Oh, it seems I understand. We can make it only for ListItem, because we know the markup. In the first comment I started to think about common solution, sorry )

Reply:
Done